docs/Consensus-Protocols/IBFT.md

```diff
@@ -77,6 +53,48 @@ Properties that have specific values in IBFT 2.0 genesis files are:
 
 To start a node on an IBFT 2.0 private network, use the [`--genesis-file`](../Reference/Pantheon-CLI-Syntax.md#genesis-file`) option to specify the custom genesis file. 
 
+### Extra Data 
+
+The `extraData` property is RLP encoded. RLP encoding is a space efficient object serialization scheme 
+used in Ethereum. Use the Pantheon subcommand [`rlp encode`](../Reference/Pantheon-CLI-Syntax.md#rlp) 
+to generate the `extraData` RLP string to include in the genesis file. 
+
+!!! example                                        
+    ```bash
+    pantheon rlp encode --from=toEncode.json --to=rlpEncodedExtraData
```

Comment:
you can also omit the --to option and simply have it displayed on the console, simpler to copy from the console as anyway you copy the value from the file in the next step.

Reply:
Fixed